zebedee/client_test.go

```diff
@@ -230,8 +230,10 @@ func TestUnitClient(t *testing.T) {
 	})
 
 	Convey("test GetBulletin", t, func() {
+		collectionId := ""
+		lang := "en"
 		Convey("returns the latest release of a bulletin", func() {
-			b, err := cli.GetBulletin(ctx, testAccessToken, "", "bulletin-latest-release")
+			b, err := cli.GetBulletin(ctx, testAccessToken, collectionId, lang, "bulletin-latest-release")
```

Comment:
not sure about the requirement but should we consider the welsh lang too for a scenario testing

Reply:
Similarly, there is nothing in the other existing tests to assert the value set as language when it reaches the mocked server but we could try and amend them if necessary?